TACC/tup-ui
# fix(tup-cms): allow use of assets_custom_delayed

## Overview

Rather than overwrite template `assets_custom_delayed.html`, include desired template into it.

## Related

- fixes bug in #148
- better than #165

## Changes

- save new template as new template
- import new template into existing template

## Testing

1. `nx serve tup-cms`
2. `nx serve tup-cms-react`
3. Add/Verify markup `<div id="cms-sysmon"></div>` on a CMS page.
4. Verify System Monitor loads.

## UI

![sysmon local load test](https://user-images.githubusercontent.com/62723358/224450824-7546c244-29cc-4eb3-8397-7d097dda6522.png)
